### ISSUE TITLE
LESYMAP v0.0.0.9201 (Release date: 2018-09-14) questions

### ISSUE THREAD

**Issue by marciekj** [dorianps/LESYMAP#28]
Hello- 

I am a new LESYMAP user and am trying to get a better grasp on what my output means. I understand that LESYMAP now allows for SCCAN weights to show a directional relationship to behavior. In the "what's new" explanation of this update you write: 

"The stat.img map produced from SCCAN will now contain correctly oriented voxel weights with respect to behavior. Negative weights are inversely related to behavioral scores (1-lesion = lower behavioral score), and positive weights are directly related to behavioral scores (0-healthy = lower behavioral score). While stat.img contains normalized voxels weights between -1 and 1, raw voxel weights can be found in rawWeights.img."

I am confused what you mean when you say "1-lesion=lower behavioral score and 0-healthy=lower behavioral score." Further explanation of this would be very useful. 

Thanks!

**Comment by dorianps** [comment]
@marciekj 
Perhaps this is not conveyed well in the wiki.

A lesioned voxel has typically a value of 1 while a non-lesioned voxel has a value of 0. You have to think of this as two extremes 0 (healthy) to 1 (damage). This binary jump is the dimension of the lesion damage. You need to think how this is linked with behavior. For example, if the behavioral scores show the deficit as a decrease, then a positive relationship/correlation with lesion would mean:

lower behavior scores are in areas where voxel values are low (i.e., 0, or healthy)

A negative link would mean the opposite:

lower behavior scores are in areas where voxel values are high (i.e., 1, or lesioned)

This would, of course, depend on the type of behavior measured. But in principle, a negative symptom (lowering of normal behavior score) normally leads to an interest in negative SCCAN weights, i.e., the areas that are negatively correlated with the lowering of that behavioral score.

Hope it's clear, and hope I am not missing something. I have not used Lesymap for a while now and some memories are getting rusty.

**Comment by marciekj** [comment]
Hi Dorian, 

Thanks for this explanation-- it definitely helped clarify some of my questions. I thought it might be useful to give you an example of my output  just to be sure I am understanding correctly. I ran LESYMAP on a group of 67 people who have focal lesions and wanted to see if there was a relationship between lesion location and different aspects of psychological well-being. I set the directionalSCCAN option in LESYMAP to "true" so that SCCAN could look for both positive and negative relationships. 

Out of the seven LESYMAP analyses I ran, two produced significant stat maps. The first result showed a relationship between lesion location and "positive relationships with other." The output was as follows: 
Range rawWeights: 0 0.000340686150593683
Number of Clusters: 1
Cluster sizes (voxels): 2729
Cluster sizes (mm3): 2729
optimalSparseness: 0.0223832498217236
CVcorrelation.stat: 0.283762508251304
CVcorrelation.pval: 0.0199631006419305

When I view the stat image there is a blob with voxel weights ranging from 0-1. Am I right to interpret this as higher scores on my behavioral measure are associated with damage in these voxels? 

The second result showed a relationship between lesion location and "purpose in life." The output was as follows: 
Range rawWeights: -6.27334229648113e-05 6.16540783084929e-05
Number of Clusters: 16
Cluster sizes (voxels): 17870 16981 14393 11629 2906 2493 2138 1497 1151 845 816 491 318 221 192 159
Cluster sizes (mm3): 17870 16981 14393 11629 2906 2493 2138 1497 1151 845 816 491 318 221 192 159
optimalSparseness: -0.570445597620631
CVcorrelation.stat: 0.339131496728208
CVcorrelation.pval: 0.00499454614143623

When I view the stat image, there are various blobs, some of which have negative weights and some of which have positive weights. Am I correct to interpret this as voxels with positive weights indicate that higher scores on my behavioral measure are associated with damage in these voxels and voxels with negative weights indicate that lower sores on my behavioral measure are associated with damage in these respective voxels?

Thank you in advance for your assistance with this! 

Best, 
Marcie

**Comment by dorianps** [comment]
> Hi Dorian,
> 
> Thanks for this explanation-- it definitely helped clarify some of my questions. I thought it might be useful to give you an example of my output just to be sure I am understanding correctly. I ran LESYMAP on a group of 67 people who have focal lesions and wanted to see if there was a relationship between lesion location and different aspects of psychological well-being. I set the directionalSCCAN option in LESYMAP to "true" so that SCCAN could look for both positive and negative relationships.
> 
> Out of the seven LESYMAP analyses I ran, two produced significant stat maps. The first result showed a relationship between lesion location and "positive relationships with other." The output was as follows:
> Range rawWeights: 0 0.000340686150593683
> Number of Clusters: 1
> Cluster sizes (voxels): 2729
> Cluster sizes (mm3): 2729
> optimalSparseness: 0.0223832498217236
> CVcorrelation.stat: 0.283762508251304
> CVcorrelation.pval: 0.0199631006419305
> 
> When I view the stat image there is a blob with voxel weights ranging from 0-1. Am I right to interpret this as higher scores on my behavioral measure are associated with damage in these voxels?

Yes, that should be correct. Positive: higher voxel value (lesion) =higher behavior score.

> 
> The second result showed a relationship between lesion location and "purpose in life." The output was as follows:
> Range rawWeights: -6.27334229648113e-05 6.16540783084929e-05
> Number of Clusters: 16
> Cluster sizes (voxels): 17870 16981 14393 11629 2906 2493 2138 1497 1151 845 816 491 318 221 192 159
> Cluster sizes (mm3): 17870 16981 14393 11629 2906 2493 2138 1497 1151 845 816 491 318 221 192 159
> optimalSparseness: -0.570445597620631
> CVcorrelation.stat: 0.339131496728208
> CVcorrelation.pval: 0.00499454614143623
> 
> When I view the stat image, there are various blobs, some of which have negative weights and some of which have positive weights. Am I correct to interpret this as voxels with positive weights indicate that higher scores on my behavioral measure are associated with damage in these voxels and voxels with negative weights indicate that lower sores on my behavioral measure are associated with damage in these respective voxels?

Yes, the conclusion is in principle correct. But in reality more complicated. There are cases of almost paradoxical findings in which you find a negative relationship with behavior. That would mean that when the area is healthy (non-lesioned) the deficit is more pronounced, which is almost paradoxical. I think some researchers went as far as stating that the preservation of that "healthy" area makes the deficit worse, which would mean that it would be better if you actually lesion that area to improve the deficit. However, the finding can be a methodlogical artifact as well (I think Chris Sperber discussed this in a paper). The problem is that lesions don't occur randomly. When area A is lesioned, area B is healthy, and vice versa. When the left hemisphere is lesioned, the right hemisphere is not (often by inclusion criteria in the study). If you study language, the analysis may pick up the opposite finding instead of the main one, i.e., instead of picking up that a lesion in the left hemisphere causes aphasia, it may pick up that a healthy right hemisphere causes aphasia, which is obviously not true. So, you need to be careful on those inverse, unexpected results, and guide your interpretation based on prior knowledge as well. Generally speaking, a direct relationship between lesion and worsening of symptoms makes more sense than symptoms that worsen when the area is healthy.

Fyi, I suspect that the reason why sometimes the inverse effect becomes more prominent is related more to statistical power. If a region has voxels more equally split 50-50 between subjects (lesioned vs. non-lesioned), the statistical model has more power than another region that is split 40-60.


> 
> Thank you in advance for your assistance with this!
> 
> Best,
> Marcie

**Comment by dorianps** [comment]
Btw, when I tried to include positive and negative relationships in the SCCAN outcome, the main idea was to allow the model to expand in both directions, because sometimes it would converge in a paradoxical inverse relationship which had more statistical power while ignoring the other (correct) side. The idea therefore was that even smaller effects in the right direction can be picked up, and the researcher has the ability to ignore inverse paradoxical result.

**Comment by marciekj** [comment]
I see, that all makes sense, I think. I guess I'm just struggling a little to make heads and tails of my result because on the behavioral measure I am using, a higher score means higher ("better") self-reported well-being. Thus, where I am seeing positive correlations (positive voxel weights), it would indicate that damage is related to higher well-being (not higher than average necessarily...this actually fits with the literature although it is a little hard to grapple with). Where I am seeing negative voxel weights (i.e., negative correlations between brain and behavior), I thought that indicates that damage to these areas are related to lower levels of well-being. Is this correct? Or is it that those regions show healthy tissue in which the deficit (i.e., low well-being) is more pronounced? Sorry, I got a bit confused by you last explanation.... Thanks!

**Comment by dorianps** [comment]
Well, I can't find out which interpretation is true, i.e., whether low well being emerges because of of preserved healthy tissue or high well being emerges because of lesioned tissue. There are neurologic conditions associated with anosognosia, in which the patient does not recognize his deficit and may think he is fine (hemispatial neglect and Alzheimers). I can also think that a lesion in brain centers related to emotion/depression can make the participant feel better. And it can be a methodological artifact as well, as I explained above. You can also run complementary univariate analyses with t-tests or BMfast to make sure the results match to some degree the SCCAN results. Well being is quite an abstract psychological construct to me, a bit far from the reproducible performance the patient can give in a parametric test.

**Comment by marciekj** [comment]
I agree, well being is quite abstract and perhaps not super well suited to this method... I think my lingering question is how to interpret the negatively weighted voxels. Would you interpret them as **healthy tissue** related to a more pronounced "deficit" in well-being OR **lesioned tissued** related to more pronounced "deficit" in well-being?

**Comment by dorianps** [comment]
If a deficit in well being is considered lowering of the scores, then a deficit in well being would be linked to a lesion (higher voxel value) where SCCAN weights are negative (again, double check with univariate LSM to make sure the direction is consistent).
